sunset/code/game/objects/structures/table_frames.dm

```diff
@@ -0,0 +1,2 @@
+/obj/structure/table_frame
+	icon = 'sunset/icons/obj/structures.dmi'
```

Comment:
Did you change the sprite?it dosent seem so

Reply:
Yes, it's very slight though, but it's a different shade.